qt/python/resources.qrc

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE RCC><RCC version="1.0">
+<qresource>
+    <file alias="MantidPlot_Icon_32offset.png">../../images/MantidPlot_Icon_32offset.png</file>
+    <file alias="data_replace.png">../../images/data_replace.png</file>
+    <file alias="fileclose.png">../../images/fileclose.png</file>
```

Comment:
I'm seeing an error saying `Cannot find file: ../../images/fileclose.png`. It looks like this file is in `MantidPlot/icons` and not `images`